modules/@apostrophecms/attachment/index.js

```diff
@@ -708,8 +708,8 @@ module.exports = {
               if (ancestorFields) {
                 value = _.clone(value);
                 o.attachment = value;
-                value._crop = _.pick(ancestorFields, 'width', 'height', 'top', 'left');
-                value._focalPoint = _.pick(ancestorFields, 'x', 'y');
+                value._crop = ancestorFields.width ? _.pick(ancestorFields, 'width', 'height', 'top', 'left') : undefined;
+                value._focalPoint = ancestorFields.x ? _.pick(ancestorFields, 'x', 'y') : undefined;
```

Comment:
x could legitimately be zero, so this should check that `typeof` is `'number'`.

Reply:
Changed.